module/VuFind/src/VuFind/Db/Entity/ExternalSessionEntityInterface.php

```diff
@@ -40,4 +42,58 @@
  */
 interface ExternalSessionEntityInterface extends EntityInterface
 {
+    /**
+     * Get identifier (returns null for an uninitialized or non-persisted object).
+     *
+     * @return ?int
+     */
+    public function getId(): ?int;
+
+    /**
+     * Get PHP session id string.
+     *
+     * @return string
+     */
+    public function getSessionId(): string;
+
+    /**
+     * Set PHP session id string.
+     *
+     * @param string $sessionId PHP Session id string
+     *
+     * @return ExternalSessionEntityInterface
+     */
+    public function setSessionId(string $sessionId): ExternalSessionEntityInterface;
+
+    /**
+     * Get PHP external session id string.
```

Comment:
The external session ID is not a PHP session, so you should remove "PHP" from the comments on the getter and setter here. (The purpose of this table is to associate the local PHP session in `session_id` with third-party session IDs in `external_session_id`).